data.sql

```diff
@@ -6,3 +6,77 @@ VALUES
   (2, 'Gabumon', '2018-11-15', 2, true, 8),
   (3, 'Pikachu', '2021-01-07', 1, false, 15.04),
   (4, 'Devimon', '2017-05-12', 5, true, 11);
+
+/* Add new animals to the 'animal' table */
+
+INSERT INTO animals (id, name, date_of_birth, escape_attempts, neutered, weight_kg)
+VALUES
+  (5, 'Charmander', '2020-02-8', 0, false, 11),
+  (6, 'Plantmon', '2022-11-15', 2, true, 5.7),
+  (7, 'Squirtle', '1993-04-02', 3, false, 12.13),
+  (8, 'Angemon', '2005-06-12', 1, true, 45),
+  (9, 'Boarmon', '2005-06-07', 7, true, 20.4),
+  (10, 'Blossom', '1998-10-13', 3, true, 17);
+
+/* Inside a transaction update the animals table by setting the species column to unspecified. Verify that change was made. Then roll back the change and verify that species columns went back to the state before transaction. */
```

Comment:
- [ ] It would be great if we don't leave comments in the final code. Therefore Please kindly take out all comments from code it is often considered good practice for clean and readable code, what